### PR TITLE
add 2 more test orgs under CareSharing vendor

### PR DESCRIPTION
**Pull request by peekaacho**

### CODE REVIEW

**Comment by peekaacho** [comment]
tagging @woutslakhorst or @stevenvegt for this
thanks! :)

File: organizations.json [hunk 1]
```diff
@@ -8,5 +8,15 @@
     "name": "CBoards Development",
     "identifier": "urn:oid:2.16.840.1.113883.2.4.6.1:12481248",
     "publicKey": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA25XlsYggeqianbWcr5z1\neC97Y3sSdVAP+KodLFvSj/kVnHl8+ATcYc9bsdTd7dcJpcrWUgjuz3WvLVbZFhEv\nragXlI+kfJNJhjy2mUE2U/eKKCDgQ2xeXDqKDr6S77fhbB+3XL1VqNPi2ShQhbBU\n8roOM+BiACQMj6vNgza8tVoHj/UmATDdFqjmr3WLR+Atg1K8wPZC3us98UCEdIqB\nQF64AfRDNzmBLMycR3pmskIrmnuF8eSE7g3xGpmrdpInA6QU4AkmUvHN36ubCS2M\nkj7h4PEq3JcPkBIDfVb/A/05PN7OZGdCjnu7FKW7glgIEkXDr3AGFuNqt8cb3YFE\n7wIDAQAB\n-----END PUBLIC KEY-----"
+  },
```
Comment (woutslakhorst) [inline organizations.json]: This file is deprecated, no need to change it. Better to remove the changes

File: endpoints_organizations.json [hunk 2]
```diff
@@ -13,5 +13,15 @@
     "status": "active",
     "organization": "urn:oid:2.16.840.1.113883.2.4.6.1:12481248",
     "endpoint": "urn:ietf:rfc:1779:O=CBoards,C=NL,L=Groenlo,CN=nuts_experimental-development-cboards"
+  },
```
Comment (woutslakhorst) [inline endpoints_organizations.json]: This file is deprecated, no need to add here, better to remove these changes.

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

**Comment by woutslakhorst** [comment]
Rest is OK: @reinkrul , do you see any problems?

**Comment by reinkrul** [comment]
> Rest is OK: @reinkrul , do you see any problems?

LGTM

I tested them against the latest versions, the events loaded OK and I was able to lookup the new organizations and endpoints through the API. One thing I noticed though is that the endpoint identifiers are all the same but that shouldn't be a problem. Endpoint.ID will be deprecated in future (and migrated to a property when necessarry) anyways.

**Review by reinkrul — APPROVED** [review]

**Comment by peekaacho** [comment]
done committing requested changes :)
 @woutslakhorst @reinkrul

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**